src/main/resources/assets/computercraft/lang/it_it.lang

```diff
@@ -0,0 +1,83 @@
+tile.computercraft:computer.name=Computer
+tile.computercraft:advanced_computer.name=Computer Avanzato
+tile.computercraft:drive.name=Lettore disco
+tile.computercraft:printer.name=Stampante
+tile.computercraft:monitor.name=Monitor
+tile.computercraft:advanced_monitor.name=Monitor Avanzato
+tile.computercraft:wireless_modem.name=Modem Wireless
+tile.computercraft:wired_modem.name=Modem Cablato
+tile.computercraft:cable.name=Cavo Di Rete
+tile.computercraft:command_computer.name=Computer Commando
+tile.computercraft:advanced_modem.name=Modem di Ender
+tile.computercraft:speaker.name=Altoparlante
+
+tile.computercraft:turtle.name=Tartaruga
+tile.computercraft:turtle.upgraded.name=Tartaruga %s
+tile.computercraft:turtle.upgraded_twice.name=Tartaruga %s %s
+tile.computercraft:advanced_turtle.name=Tartaruga Avanzata
+tile.computercraft:advanced_turtle.upgraded.name=Tartaruga %s Avanzata
+tile.computercraft:advanced_turtle.upgraded_twice.name=Tartaruga %s %s Avanzata
+
+item.computercraft:disk.name=Disco Floppy
+item.computercraft:treasure_disk.name=Disco Floppy
+item.computercraft:page.name=Pagina Stampata
+item.computercraft:pages.name=Pagine Stampate
+item.computercraft:book.name=Libro Stampato
+
+item.computercraft:pocket_computer.name=Computer Tascabile
+item.computercraft:pocket_computer.upgraded.name=Computer Tascabile %s
+item.computercraft:advanced_pocket_computer.name=Computer Tascabile Avanzato
+item.computercraft:advanced_pocket_computer.upgraded.name=Computer Tascabile %s Avanzato
+
+upgrade.minecraft:diamond_sword.adjective=Da Combattimento
```

Comment:
I have 0 Italian knowledge so this may be a dumb question, but just want to double-check these work with all forms of Turtle & Pocket computer? I know other languages with genders sometimes have to jump through hoops to get something consistent.

Reply:
I thought about it and adapted the translations